deploy/deploy.yaml

```diff
@@ -72,6 +73,9 @@ spec:
             - mountPath: /kinitiras/log
               name: log
               subPathExpr: $(POD_NAME)
+            - mountPath: /etc/templates
```

Comment:
Is this needed?

Reply:
Not any more, add this for more convenient for debug and test, I'll remove it.